draft-terbu-sd-jwt-vc.md

```diff
@@ -524,68 +524,108 @@ model defined in [@VC-DATA], this specification defines a mapping algorithm for
 VC-SD-JWT and VP-SD-JWT to the vocabulary and data model defined W3C VCDM 2.0
 which is based on JSON-LD.
 
-### VC Directory
+### W3C VC Specifications Directory
 
 This specification registers the media type `application/vc+sd-jwt` in the
-W3C Verifiable Credentials (VC) Directory.
+W3C Verifiable Credentials (VC) Directory [@VC-DIR].
 
 ### Mapping Algorithm
 
 The following is a uni-directional transformation algorithm that takes in a
 VC-SD-JWT conformant to this specification and maps it
-to the corresponding properties in the W3C VCDM 2.0 [VCDM2.0]
+to the corresponding properties in the W3C VCDM 2.0 [@VC-DATA]
 which is based on a JSON-LD vocabulary. It includes specific handling for JWT
 claims used in this specification. The function returns a Verifiable
 Credential object in the W3C VCDM 2.0 format.
 
+Procedure:
+1. Let `payload` be the unsecured payload of the VC-SD-JWT reconstructed from the SD-JWT and Disclosures.
```

Comment:
```suggestion
Procedure:
1. Validate SD-JWT following the steps defined in Section 6.2 of [@oauth-selective-disclosure-jwt].
1. Let payload be the payload of the VC-SD-JWT obtained in the previous step.
```

I thought we are calling it SD-JWT-VC and not VC-SD-JWT..?
I would prefer we move away from a VC WG specific terminology like secure/unsecured - signed, unsigned is simpler, no?
There is no `payload` claim in VCDM, right..?

Reply:
Can we please have the naming discussion in a separate issue? This PR does not introduce a new name, only the algorithm.

Reply:
Agree with @Sakurann's proposals but to move naming discussion elsewhere.

Reply:
I created this issue https://github.com/vcstuff/draft-terbu-sd-jwt-vc/issues/70, resolving this discussion.